package.json

```diff
@@ -69,6 +69,7 @@
     "packages/remix",
     "packages/replay",
     "packages/replay-worker",
+    "packages/rollup-utils",
```

Comment:
Can we just make this `build`? If we move to a different package, we shouldn't change the package name

Reply:
I am gonna leave it at `rollup-utils` because that is what it is. `build` is too generic since building has many more steps than just calling some rollup utils.